locations/spiders/au_vide_grenier_fr.py

```
@@ -0,0 +1,16 @@
+
+from locations.categories import Categories
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class AuVideGrenierFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "au_vide_grenier_fr"
+    item_attributes = {
+        "brand": "Au Vide Grenier",
+        "brand_wikidata": "Q141175815",
+        "extras": Categories.SHOP_SECOND_HAND.value,
+    }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)